tests/Rule/LatteTemplatesRule/PresenterWithoutModule/CollectorResultForPresenterTest.php

```diff
@@ -45,10 +45,10 @@ public function testComponents(): void
             __DIR__ . '/Source/SomeTableControl.php',
         ], [
             'NODE NetteApplicationUIPresenter {"className":"\PresenterWithoutModule\Fixtures\ComponentsPresenter"}',
-            'TEMPLATE default.latte ComponentsPresenter ["startupParent","presenter","variableFromParentCalledViaParent"] ["form","parentForm","parentForm","onlyParentDefaultForm","someControl"]',
-            'TEMPLATE create.latte ComponentsPresenter ["startupParent","presenter"] ["form","parentForm","onlyCreateForm"]',
-            'TEMPLATE parent.latte ComponentsPresenter ["startupParent","presenter","variableFromParentAction"] ["form","parentForm","parentDefaultForm"]',
-            'TEMPLATE noAction.latte ComponentsPresenter ["startupParent","presenter"] ["form","parentForm"]',
+            'TEMPLATE default.latte ComponentsPresenter ["startupParent","presenter","variableFromParentCalledViaParent"] ["form","noType","parentForm","parentForm","onlyParentDefaultForm","someControl"]',
```

Comment:
This looks really complicated :)

Reply:
It is but it is simplest yet specific enough I can think of.